### ISSUE TITLE
Look into increasing contrast of the conditional view

### ISSUE THREAD

**Issue by Kapin** [SanaMobile/sana.protocol_builder#396]
<img width="775" alt="screen shot 2016-03-06 at 3 19 56 pm" src="https://cloud.githubusercontent.com/assets/995382/13556724/ee82641a-e3ae-11e5-8339-622c29ce8bf2.png">

Not sure the best way to handle, but it can be hard to follow with maximum nesting. Maybe shade the inner levels?

**Comment by Trinovantes** [comment]
I don't have time to experiment with designs.
